src/index.ts

```diff
@@ -309,16 +309,17 @@ class TranscriptionEditor {
         const annotation = annotationBlock.annotation;
         const editorContent = window.tinymce.get(annotationBlock.editorId).getContent();
         // add the content to the annotation
-        annotation.motivation = "supplementing";
         if (Array.isArray(annotation.body) && annotation.body.length == 0) {
             annotation.body.push({
                 type: "TextualBody",
-                purpose: "transcribing",
                 value: editorContent || "",
                 format: "text/html",
                 label: annotationBlock.labelElement.textContent || undefined,
-                // TODO: transcription motivation, language, etc.
+                // purpose: "transcribing",
+                // - purpose on body is only needed if more than one body
+                //   (e.g., transcription + tags in the same annotation)
             });
+            // TODO: transcription language, etc.
```

Comment:
FWIW, I decided at some point that we shouldn't duplicate transcription language in the annotations, since we're already tracking that in the PGP document metadata.